src/app/[locale]/(home)/layout.tsx

```
@@ -1,12 +1,23 @@
+import Footer from "@components/layout/footer";
 import Header from "@components/layout/header";
-import React from "react";
+import { Fragment, ReactNode } from "react";
```

ESLint

[error] 'Fragment' is defined but never used.

(@typescript-eslint/no-unused-vars)